.devcontainer/Dockerfile.console

```diff
@@ -0,0 +1,19 @@
+FROM openshift/origin-cli:latest AS cli
+FROM quay.io/coreos/tectonic-console-builder:v23 AS build
+SHELL ["/bin/bash", "-c"]
+
+COPY --from=cli /usr/bin/oc /usr/local/bin/oc
+RUN git clone https://github.com/openshift/console /console
+
+WORKDIR /console
+RUN [ ! -f "./bin/bridge" ] && ./build-backend.sh
+RUN [ ! -d "./frontend/public/dist" ] && ./build-frontend.sh
+
+ENV OC_URL=$OC_URL
+ENV OC_PASS=$OC_PASS
+ENV OC_USER=$OC_USER
+ENV OC_PLUGIN_NAME=$OC_PLUGIN_NAME
+
+CMD eval "oc login $OC_URL -u $OC_USER -p $OC_PASS --insecure-skip-tls-verify" && \
+    source ./contrib/oc-environment.sh && \
+    ./bin/bridge -plugins $OC_PLUGIN_NAME=http://localhost:9001 --k8s-mode-off-cluster-thanos=$(oc -n openshift-monitoring get configmap sharing-config -o jsonpath='{.data.prometheusURL}')
```

Comment:
Why do you need to set `--k8s-mode-off-cluster-thanos`? I'd expect that to be handled by `oc-environment.sh`.